src/environments/large_scale_lunar.py

```diff
@@ -168,6 +168,7 @@ def add_robot_manager(self, robotManager: RobotManager) -> None:
         """
 
         self.robotManager = robotManager
+        self.robotManager.start_TMTC()
```

Comment:
should the TMTC always be started or should there be an if here?
e.g if the yaml config does NOT specify yamcs_tmtc then the user doesnt want this feature activated

same for the other environments